deployments/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+typescript@^5.0.4:
+  version "5.0.4"
+  resolved "https://registry.npmjs.org/typescript/-/typescript-5.0.4.tgz"
+  integrity sha512-cW9T5W9xY37cc+jfEnaUvX91foxtHkza3Nw3wkoF4sSlKn0MONdkdEndig/qPBWXNkmplh3NzayQzCiHM4/hqw==
```

Comment:
Do we really need this file in the repo?

Reply:
Weeeeell, we started off with - let's have an API and turned 180° degrees to - let's just include .json files. I'm including the json files through the tsc binary - which transforms ts to js. We don't have any ts files right, but I use it to copy 
the "./artifacts/**/*.json" and "./abis/**/*.json" files to the dist folder. 

If we don't want to use tsc for this, then I could have a bash script that creates the dist folder and copies stuff to it.

Reply:
I mean, I was just wondering whether we could remove the auto-generated stuff from the repo -- because we don't really need it.

I think the "bash script" you mentioned could easily be just a new Makefile target where you could say e.g.

```
make npm-deployments-package
```
(or whatever name you prefer)
and it would take care to do all of that stuff.